nixos/infrastructure/flyingcircus-virtual.nix

```diff
@@ -83,12 +83,24 @@ mkIf (cfg.infrastructureModule == "flyingcircus") {
       fsType = "xfs";
     };
     "/tmp" = {
-      device = "/dev/disk/by-label/tmp";
+      device = "/dev/disk/by-partlabel/tmp";
       fsType = "xfs";
       noCheck = true;
     };
   };
 
+  flyingcircus.initrd = {
+    upgradeXFS = {
+      "/dev/disk/by-label/root" = [ "bigtime=1" ];
+    };
+    # XXX: remove nrext=0 once all VMs are on kernel 6.6+
```

Comment:
This confuses me. As 25.05 uses Kernel 6.6+,  we should be able to use this here. Also the requirement _all VMs_ is not necessary anymore in this code path